### PR TITLE
Minification bug

### PR DESCRIPTION
**Pull request by joeatwork**
Test for a minification bug, where the closure compiler would alias "Token" to "ad". Includes a test for this specific error site.

### CODE REVIEW

**Comment by joeatwork** [comment]
@tdumitrescu @ttrefren @jbwyme or somebody please take a peek. This is an urgent fix and will go out ASAP

File: tests/test.js [hunk 1]
```diff
@@ -266,6 +266,25 @@ mpmodule("mixpanel.track");
         });
     });
 
+    asyncTest("check no property name aliasing occurs during minify", 1, function() {
```
Comment (jbwyme) [inline tests/test.js]: i think we can just test that we can override the token with a new token in the properties object

**Comment by tdumitrescu** [comment]
Ship it

Should later audit the rest of the public-facing funcs for similar potential closure-compiler timebombs

File: tests/test.js (hunk 1 above)
Comment (joeatwork) [inline tests/test.js]: Added a new test for token override here: bd749ab and left this test (which will detect 'ad' and 'ab' and other minified name bugs)

**Comment by joeatwork** [comment]
@tdumitrescu you are the man- thanks for your help during your time off!

**Comment by tdumitrescu** [comment]
awesome @joeatwork thanks for jumping on this

**Pull request merged**